students/jinsup/westagram/users/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import json, re
+from django.http import JsonResponse
+from django.views import View
+from users.models import User
 
-# Create your views here.
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+
+            if re.match(r"^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$", data["email"]) == None:
+                return JsonResponse({'message': 'INVALID_EMAIL'}, status=400)
```

Comment:
```suggestion
            if not re.match(r"^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$", data["email"]):
                return JsonResponse({'message': 'INVALID_EMAIL'}, status=400)
```
None이 False로 인식 되기 때문에 다음과 같이 수정하는 것이 좋습니다.